packages/longhorn/generated-changes/patch/questions.yml.patch

```diff
@@ -48,13 +48,22 @@
 @@ -71,7 +71,7 @@
      label: Longhorn Share Manager Image Tag
      group: "Longhorn Images Settings"
+   - variable: image.longhorn.backingImageManager.repository
+-    default: longhornio/backing-image-manager
++    default: rancher/longhornio-backing-image-manager
```

Comment:
I think all the images in this file should default to the `mirrored-` ones. I'm curious if this default has been causing people to have problems with airgap. Whatever is in values.yaml is what gets added to airgap images.txt.